pilot/pkg/proxy/envoy/v2/debug.go

```diff
@@ -611,6 +612,22 @@ func (s *DiscoveryServer) ConfigDump(w http.ResponseWriter, req *http.Request) {
 	w.Write([]byte("You must provide a proxyID in the query string"))
 }
 
+// PushStatusHandler dumps the last PushStatus
+func (s *DiscoveryServer) PushStatusHandler(w http.ResponseWriter, req *http.Request) {
+	if model.LastPushStatus == nil {
```

Comment:
Do we need to lock around accessing the push status?

Reply:
The object is going to be modified while the push is going, but once it gets copied to LastPushStatus it no loger
gets modified. 

The pointer itself may change - and that may need a lock.